### PR TITLE
doc: vastly extended by adding method documentation

### PR DESCRIPTION
**Pull request by Alex-Werner**
### Issue being fixed or implemented  

This is a follow-up of https://github.com/dashevo/DashJS/pull/17. 
We added improved greatly the coverage of the documentation especially by adding full method documentation for our main types Account, Wallet, Keychain, Storage,...

### What was done  

- Doc : Account
- Doc : Keychain
- Doc : Storage
- Doc : Wallet
- Doc : Events

### CODE REVIEW

File: docs/account/createTransaction.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+| **change**             | string                        | no                           | Defaults: `account.getUnusedAddress(internal)`. When set, will use that address as a change address on remaining fund               |
+
+
+Returns : [Transaction](https://dashevo.github.io/DashJS/#/usage/dashcorelib-primitives?id=transaction)   
+Notes: This transaction will be need to be signed `account.sign(transaction)` and then, if wanted, broadcasted to the network for execution `account.broadcastTransaction()`.
+
+Example : 
+```js
+const recipient = "yereyozxENB9jbhqpbg1coE5c39ExqLSaG";
+const satoshis = 10e8;
```
Comment (thephez) [inline docs/account/createTransaction.md]: Typo ("10e8")?

File: docs/account/decrypt.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+|-------------------|--------|----------------| -------------------------------------------------|
+| **method**        | String | yes            | Enter a valid decrypt method (one of: ['aes']) |
+| **data**          | String | yes            | An encrypted value                                  |
+| **secret**        | String | yes            | The secret used for encrypting the data in first place     |
+
+Returns : decoded value (string).
+
+```js
+const decrypted = account.decrypt('aes','U2FsdGVkX18+7ixRbZ7DzC8P8X/4ewNHSp2R6pZDmsI=', 'secret')
+console.log(decrypted);// coucou
```
Comment (thephez) [inline docs/account/decrypt.md]: Unnecessary comment?
```suggestion
console.log(decrypted);
```

Comment (thephez) [reply docs/account/decrypt.md]: Oh, I see. This is the value shown in the encrypt document :+1:

File: docs/account/forceRefreshAccount.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+**Usage**: `account.forceRefreshAccount()`    
+**Description**: Force a refresh of all the addresses informations (utxo, balance, txs...) by invalidating previous.   
+**Important**: This is used for experience developer. If you have an issue or need using it, please do not hesitate to post an issue on GitHub.   
+
+Parameters:
```
Comment (thephez) [inline docs/account/forceRefreshAccount.md]: Looks like the following info is for the encrypt method

File: docs/account/getUTXOS.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+**Usage**: `account.getUTXOS(onyAvailable)`
```
Comment (thephez) [inline docs/account/getUTXOS.md]: Parameter section below uses "onlySpendable" (vs the "onlyAvailable" used here). Also, missing the "l" in only.

File: docs/account/getUnconfirmedBalance.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+**Usage**: `account.getUnconfirmedBalance(displayDuffs)`      
+**Description**: This method will return the unconfirmed balance (missing inclusion in a block) of an accout
```
Comment (thephez) [inline docs/account/getUnconfirmedBalance.md]:
```suggestion
**Description**: This method will return the unconfirmed balance (missing inclusion in a block) of an account
```

File: docs/events/fetched_confirmed_transaction.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+**Usage**: `account.events.on('FETCHED/CONFIRMED_TRANSACTION', fn)`    
+**Description**: Every time a new unconfirmed transaction is fetched from the network, an event in thrown.
```
Comment (thephez) [inline docs/events/fetched_confirmed_transaction.md]:
```suggestion
**Description**: Every time a new confirmed transaction is fetched from the network, an event in thrown.
```

File: docs/events/unconfirmed_balance_changed.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
+**Usage**: `account.events.on('unconfirmed_balance_changed', fn)`    
+**Description**: Wallet-lib, when finished to perform it's internal tasks (blockheight, SPV, utxos sync...), will throw this event.
```
Comment (thephez) [inline docs/events/unconfirmed_balance_changed.md]: Description from a different method

File: docs/account/createTransaction.md (hunk 1 above)
Comment (Alex-Werner) [reply docs/account/createTransaction.md]: Exactly, 10e8 as in 10^8 (which is a XOR when written this way in js)

File: docs/account/forceRefreshAccount.md (hunk 3 above)
Comment (Alex-Werner) [reply docs/account/forceRefreshAccount.md]: Good catch!

**Pull request merged**